### PR TITLE
Use 0 degC instead of STDCOND for default WTEMP in TEMP option

### PR DESCRIPTION
**Pull request by vkip**

### CODE REVIEW

File: opm/input/eclipse/EclipseState/Runspec.hpp
Comment (totto82) [inline opm/input/eclipse/EclipseState/Runspec.hpp:563]: The parameters should be initialized with {false}

**Review by totto82 — CHANGES_REQUESTED** [review]
Thanks for the PR. Assuming 0 C is a good default this can be merged after the nit-pick is fixed.

File: opm/input/eclipse/EclipseState/Runspec.cpp
Comment (vkip) [reply opm/input/eclipse/EclipseState/Runspec.cpp:777]: Took you quite literally, which maybe wasn't the intention, so feel free to suggest a formulation :)

**Comment by vkip** [comment]
jenkins build this please

**Comment by vkip** [comment]
jenkins build this please

File: opm/input/eclipse/Schedule/Well/Well.cpp
Comment (totto82) [inline opm/input/eclipse/Schedule/Well/Well.cpp:1780]: This warning is given every time inj_temperature() is called. As a result the .PRT file will be full of these warnings. Is that intended?

**Comment by totto82** [comment]
When I try to run this in parallel it gets stuck in the setup. I didn't notice any think wrong/missing. I can investigate a bit more on my side to give a proper feedback. Have you tested this in parallel?

File: opm/input/eclipse/Schedule/Well/Well.cpp
Comment (vkip) [reply opm/input/eclipse/Schedule/Well/Well.cpp:1780]: Yes, I would like to give a good push to explicitly set the temperature (a simple WTEMP '*' would suffice), without actually failing if forgotten.

**Comment by vkip** [comment]
> When I try to run this in parallel it gets stuck in the setup. I didn't notice any think wrong/missing. I can investigate a bit more on my side to give a proper feedback. Have you tested this in parallel?

Did you rebuild opmgrid?

**Comment by totto82** [comment]
I am still not able to run this in parallel. Even spe1 get stuck during startup. Only when I revert all code related to default_well_inj_temperature it starts working. Since Jenkins is happy I assume this is a local problem. @akva2 Any clue?

**Comment by akva2** [comment]
I tried it on my machine now and ran various cases just fine. maybe you are building this branch and not this branch rebased on master?

**Comment by vkip** [comment]
For what it's worth, I've had this issue before when adding a member to the Well class - got stuck in loadBalance due to a throw on one proc. Making sure to rebuild opmcommon, opmgrid and then the simulator in the correct order resolved it..

**Comment by akva2** [comment]
unrelated, but that's why all developers really should use the super build and not the sibling builds. then you can't get desynced as changes are tracked across modules.

**Comment by totto82** [comment]
I final thing before we can merge this. 

The default_well_inj... member is missing from 
bool Well::operator==(const Well& data) const {

}

**Comment by vkip** [comment]
> The default_well_inj... member is missing from bool Well::operator==(const Well& data) const {

Ah.. fixed!

**Comment by vkip** [comment]
jenkins build this please

**Comment by vkip** [comment]
Thanks!

**Pull request merged**
